src/policy/lockfreeimmortalspace.rs

```diff
@@ -18,6 +18,10 @@ use crate::vm::*;
 use std::marker::PhantomData;
 use std::sync::atomic::{AtomicUsize, Ordering};
 
+/// This type implements a lock free version of the immortal collection
+/// policy. This is close to the OpenJDK's epsilon GC.
+/// Different from the normal ImmortalSpace, this version should only
+/// be used by NoGC plan, and it assumes to take up the rest of the heap range.
```

Comment:
I don't quite understand the last sentence.
Does it mean that this policy assumes it can take up the whole heap range?

Reply:
Yes. I assume so. You can take a look at its init function.